src/modal/modalcomponents/ModalContentButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { MouseEventHandler, ReactNode } from "react";
+import "../../css/ModalContentButton.css";
+
+interface ModalContentButtonProps {
+  children?: ReactNode;
+  onClick?: MouseEventHandler<HTMLElement>;
```

Comment:
P1: 아래와 같은 방식으로 작성하셔야 어떤 태그에서 이벤트가 발생되었는지 더 알 수 있습니다.

```suggestion
  onClick?: MouseEventHandler<HTMLButtonElement>;
```